model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v2/tp_model.py

```diff
@@ -151,6 +152,14 @@ def generate_tp_model(default_config: OpQuantizationConfig,
     # this configuration will be used for the operation quantization:
     default_configuration_options = tp.QuantizationConfigOptions([default_config])
 
+    # const configuration is similar to the default eight bit configuration except for PoT
+    # quantization method for the constant.
+    const_config = default_config.clone_and_edit(
+        default_weight_attr_config=default_config.default_weight_attr_config.clone_and_edit(
```

Comment:
don't you think that the OpQuantizationConfig should be defined under get_op_quantization_configs like the other op configs and the the QuantizationConfigOptions should be defined here similarly to default_configuration_options ?